packages/flutter_tools/lib/src/preview_device.dart

```diff
@@ -114,7 +106,7 @@ class PreviewDevice extends Device {
        _fileSystem = fileSystem,
        _bundleBuilderFactory = builderFactory,
        _artifacts = artifacts,
-       super('preview', ephemeral: false, category: Category.desktop, platformType: PlatformType.custom);
+       super('preview', ephemeral: false, category: Category.desktop, platformType: PlatformType.windows);
```

Comment:
It does say Windows here so now I'm a little confused. Is the JSON you posted above maybe out-of-date?

Reply:
Yes, please see the test expectation I added for more up to date JSON